doc/introduction.rst

```diff
@@ -0,0 +1,103 @@
+Introduction
+============
+
+Simpleline is a text UI framework. Originally a part of the Anaconda installer project.
+
+This UI is simple and easy to use. It is designed to be used with line-based machines and tools
+(e.g. serial console) so that every new line it appended to the bottom of the screen. Printed lines
+are never rewritten!
+
+How to use
+----------
+
+The best learning sources can be found in the
+`examples directory <https://github.com/rhinstaller/python-simpleline/tree/master/examples>`_ in the
+`GitHub repository <https://github.com/rhinstaller/python-simpleline>`_ and you can read the
+:ref:`guide_label` section of this documentation. However, the basic usage of the Simpleline will be
+shown here too to get the idea how Simpleline works::
+
+    from simpleline import App
+    from simpleline.render.screen import UIScreen
+    from simpleline.render.screen_handler import ScreenHandler
+    from simpleline.render.widgets import TextWidget
+
+
+    # UIScreen is the main building item for Simpleline. Every screen
+    # which will user see should be inherited from UIScreen.
+    class HelloWorld(UIScreen):
+
+        def __init__(self):
+            # Set title of the screen.
+            super().__init__(title=u"Hello World")
+
+        def refresh(self, args=None):
+            # Fill the self.window attribute by the WindowContainer and set screen title as header.
+            super().refresh()
+            widget = TextWidget("Body text")
+            self.window.add_with_separator(widget)
+
+
+    if __name__ == "__main__":
+        # Initialize application (create scheduler and event loop).
+        App.initialize()
+
+        # Create our screen.
+        screen = HelloWorld()
+
+        # Schedule screen to the screen scheduler.
+        # This can be called only after App.initialize().
+        ScreenHandler.schedule_screen(screen)
+
+        # Run the application. You must have some screen scheduled
+        # otherwise it will end in an infinite loop.
+        App.run()
+
+The output from the simple *Hello World* example above::
+
+    $ ./run_example.sh 00_basic
+    ================================================================================
+    ================================================================================
+    Hello World
+
+    Body text
+
+    Please make a selection from the above ['c' to continue, 'q' to quit, 'r' to
+    refresh]:
+
+In case user will press **r** and then **enter** to refresh the same screen is printed again.
```

Comment:
`In case user will press **r** and then **enter** to refresh the same` -> `If a user presses **r** and then **enter** to refresh, the same`